edweiss-app/__test__/app/(app)/lectures/remotecontrol/index.test.tsx

```diff
@@ -103,16 +89,11 @@ jest.mock('@react-native-firebase/messaging', () => ({
     })),
 }));
 
-
-
-
 // Mock react-native-pdf to prevent native calls
```

Comment:
We might need to remove this, as we are not displaying any `pdf` within the remote control 😉.

Reply:
Yup it is true. I will remove this and re-test to see if everything works. I agree that we should always have a clean, short code at the end :D

Reply:
Great 😁!